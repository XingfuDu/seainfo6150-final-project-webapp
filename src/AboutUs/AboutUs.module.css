.wrapper {
	width: 100%;
	min-height: 100%;
	background-color: #f2e9e4;
	padding: 15px;
}
.article {
	min-height: 100%;
	font-family: Arial, Helvetica, sans-serif;
	font-size: 150%;
	width: 40%;
	padding-left: 30%;
}

.h3 {
	color: #ee6c4d;
}

.p {
	color: #3d5a80;
}
