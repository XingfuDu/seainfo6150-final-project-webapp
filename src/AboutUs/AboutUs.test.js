import React from 'react';
import AboutUs from './AboutUs';
import { render } from "@testing-library/react";

describe('AboutUs tests', () => {
	it('renders correctly', () => {
		const { container } = render(<AboutUs />);
		expect(container).toMatchSnapshot();
	});
});
