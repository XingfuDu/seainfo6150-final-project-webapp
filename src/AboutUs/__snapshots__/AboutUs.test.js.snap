// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AboutUs tests renders correctly 1`] = `
<div>
  <div
    class="wrapper"
  >
    <article
      class="article"
    >
      <h3
        class="h3"
      >
        Who We Are
      </h3>
      <p
        class="p"
      >
        Book Store is the world’s largest site for readers and book recommendations. Our mission is to help people find and share books they love. Book store launched in January 2007.
      </p>
      <h3
        class="h3"
      >
        A Few Things You Can Do On Book stores
      </h3>
      <p
        class="p"
      >
        See what books your friends are reading. Track the books you're reading, have read, and want to read. Check out your personalized book recommendations. Our recommendation engine analyzes 20 billion data points to give suggestions tailored to your literary tastes. Find out if a book is a good fit for you from our community’s reviews.
      </p>
      <h3
        class="h3"
      >
        A Message From Our Co-Founder
      </h3>
      <p
        class="p"
      >
        When I was in second grade, I discovered the Hardy Boys series. Ever since, I've loved to read — both for fun and to improve my mind. And I'm always looking for the next great book.
      </p>
      <p
        class="p"
      >
        One afternoon while I was scanning a friend's bookshelf for ideas, it struck me: when I want to know what books to read, I'd rather turn to a friend than any random person or bestseller list.
      </p>
      <p
        class="p"
      >
        So I decided to build a website – a place where I could see my friends' bookshelves and learn about what they thought of all their books. Elizabeth, my co-founder (and now my wife) wrote the site copy and I wrote the code. We started in my living room, motivated by the belief that there was a better way to discover and discuss good books, and that we could build it.
      </p>
      <p
        class="p"
      >
        Book store is that site. It is a place where you can see what your friends are reading and vice versa. You can create "bookshelves" to organize what you've read (or want to read). You can comment on each other's reviews. You can find your next favorite book. And on this journey with your friends you can explore new territory, gather information, and expand your mind.
      </p>
      <p
        class="p"
      >
        Knowledge is power, and power is best shared among readers.
      </p>
      <p
        class="p"
      >
        Xingfu Du
      </p>
      <p
        class="p"
      >
        Co-Founder
      </p>
    </article>
  </div>
</div>
`;
